.github/workflows/urlchecker.yml

```diff
@@ -10,6 +10,6 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - name: urls-checker
-      uses: urlstechie/urlchecker-action@0.2.3
+      uses: urlstechie/urlchecker-action@0.0.27
```

Comment:
First of all, thanks a lot @vsoch for your pull request, I really appreciate you taking the time to not break my workflow.

The versioning of this action is... wild.

Why the jump from 0.2.3 to 0.2.31 to 0.0.25 and then eventually 0.0.27?

Reply:
It was just stupid - so the underlying software is urlchecker-python, and the original versioning scheme was totally random (not done by me, but my collaborator on the project). Since we always release them side by side, what I would have from the getgo is to have the versions match. So thinking that it wouldn't be an issue I made the (now somewhat regretful) decision to go backwards to match it. I didn't anticipate dependabot going around and asking people to update to an older version.

Reply:
Thanks for the info!

Good to have this on record, maybe this is good material for a future installment of a talk I did a while ago... 😝